test/provisioner/provision.go

```diff
@@ -64,6 +74,98 @@ type InstallOverlay interface {
 	Edit(ctx context.Context, cfg *envconf.Config, properties map[string]string) error
 }
 
+func runCommand(command string, stdout io.Writer, stderr io.Writer, args ...string) error {
+	cmd := exec.Command(command, args...)
+	cmd.Stdout = stdout
+	cmd.Stderr = stderr
+
+	fmt.Printf("Running command: %s %v\n", command, args)
+
+	if err := cmd.Run(); err != nil {
+		err = fmt.Errorf(fmt.Sprintf("Error running command: %s %v - %s", command, args, err))
+
+		log.Errorf("%v", err)
+		return err
+	}
+
+	return nil
+}
+
+func NewKeyBrokerService(clusterName string) (*KeyBrokerService, error) {
+	// Clone kbs repo
+	repoURL := "https://github.com/confidential-containers/kbs"
+
+	if err := runCommand("git", os.Stdout, os.Stderr, "clone", repoURL); err != nil {
+		return nil, err
+	}
```

Comment:
I don't think we call git to clone the repo in the code. it's hardcoded to a repo and branch and it might fail in weird ways locally if you have some auth hooks set up.

if we need the kustomization files we should clone the repo on the caller site, next to the checkout of CAA.